apps/site/assets/ts/components/Accordion.tsx

```diff
@@ -19,15 +19,13 @@ export const AccordionNoJS = ({
   children: ReactElement<HTMLElement>;
 }): ReactElement<HTMLElement> => (
   <div className="c-accordion-ui">
-    <div role="heading" aria-level={3}>
```

Comment:
I removed this div, because there doesn't appear to be a heading/title in the NoJS accordion